docs/fastdds/api_reference/includes/dds_pim/core/status/livelinessloststatus.rst

```diff
@@ -0,0 +1,7 @@
+.. _api_pim_rtpslivelinessloststatus:
+
+RTPSLivelinessLostStatus
+------------------------
+
+.. doxygentypedef:: eprosima::fastrtps::LivelinessLostStatus
+    :project: Fast RTPS
```

Comment:
This is a fastrtps status, so it is not needed to include it

Reply:
done